src/add-label.js

```diff
@@ -0,0 +1,45 @@
+const core = require('@actions/core')
+
+/**
+ * createNewPullRequest
+ * --------------------
+ * Create a new pull request inside your repository based to params passed to this function.
+ *
+ * @param {Instance} octokit Instance of octokit library
+ * @param {Object} params Object with params to filter pulls list
+ * @property {String} params.owner Repository owner user name
+ * @property {String} params.repo Repository name
+ * @property {String} params.body Pull request body content
+ * @property {String} params.title Pull request title
+ * @property {String} params.label Pull request label
+ * @property {String} params.head Branch where you want to pull the new code
+ * @property {String} params.base Branch you want to update
+ *
+ * @returns {Object} Object with opened pull request or undefined.
+ */
+async function addLabel ({octokit, params, prNumber}) {
```

Comment:
I know the parameter name is well described, but I think it would be great to have it in the JSDoc with a brief description.

Reply:
Yeah, in fact I omitted few more things in this particular JSDoc.